src/network/websockets/server/index.ts

```diff
@@ -53,6 +53,22 @@ export function runWebsocketServer<AuthDetails>(params: Params<AuthDetails>) {
       socket.send(JSON.stringify(obj));
     };
 
+    let preAuthMessageQueue: string[] = [];
```

Comment:
Question: How is cleanup for the queue happening when the connection closes? Is it handled automatically because it's defined in the `server.on("connection"` block which will fall out of scope and be garbage collected when the connection closes?

Reply:
That’s our workflow, yes. It is true that it’s easy to leave a link to an object dangling, and because everything’s here connected, it would all stay. (Some people decide not even to bother with memory leaks, just automatically restart processes when they gather too much memory. */me ducks*)

Reply:
>restart processes when they gather too much memory

Please no. :see_no_evil: I agree with you.